### ISSUE TITLE
Failed to apply mutation error on version 3.2

### ISSUE THREAD

**Issue by rahultaing** [scylladb/scylla#6254]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

*Installation details*
Scylla version (or git commit hash): 3.2.4-0.20200325.4c38534f75
Cluster size:3 
OS (RHEL/CentOS/Ubuntu/AWS AMI):debian 9

logs are filled with error message like this: 

[shard 1] storage_proxy - Failed to apply mutation from #1: std::bad_alloc (std::bad_alloc)\n

reads and writes from clients start to fail. Let me know if there is any other information i can share to root cause this issue.

**Comment by rahultaing** [comment]
we nuked all data files and restarted service on node. hasn't reproduced since then. but will update when it reproduces itself.

**Comment by dorlaor** [comment]
Worth to check for large partitions:
https://docs.scylladb.com/troubleshooting/large_partition_table/

**Comment by rahultaing** [comment]
yes i do see we have some large partitions. that link doesn't say that one of the symptom of a large partition is a failed write. and cluster being unhealthy (from clients perspective) i.e. client is unable to read/write when this happens.

**Comment by dorlaor** [comment]
It's just a guess. You had memory allocation failure and it could be due
to large partitions or large cells. There is no enough info about the schema,
the amount of mem per shard, grafana metrics of the memory and queue
snapshots

**Comment by slivne** [comment]
@rahultaing if you hit this again, it would help to get a coredump https://docs.scylladb.com/troubleshooting/report_scylla_problem/#core-dump
